pkg/minikube/cluster/cluster.go

```diff
@@ -147,6 +147,29 @@ func GetHostStatus(api libmachine.API) (string, error) {
 	return s.String(), err
 }
 
+// GetLocalkubeStatus gets the status of localkube from the host VM.
+func GetLocalkubeStatus(api libmachine.API) (string, error) {
+	host, err := checkIfApiExistsAndLoad(api)
+	if err != nil {
+		return "", err
+	}
+	s, err := host.RunSSHCommand(localkubeStatusCommand)
+	if err != nil {
+		return "", err
+	}
+	if s == "" { // this is a test pass through, I am not sure what I should do here
```

Comment:
Sorry, just noticed this one. Are you saying the SSH mock sometimes returns "" during tests? Could we change it to always return a valid string?